swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
What would happen here if the `condition` were _not_ in the expected range? Or since `condition` is a float, what would happen if it were something like `3.5`? What we could we do to handle these cases?